<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainscene</class>
 <widget class="QWidget" name="mainscene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainscene</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="GameTitleLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tetris</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="mainSceneBtnWidget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="startGameBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
